packages/ignition/src/statics-generator/utils.ts

```diff
@@ -18,7 +18,9 @@ export const write = async (
   filename: string,
   content: string
 ): Promise<void> => {
+  console.log(`Writing to ${filename}`);
```

Comment:
Let's remove it for while and afterwards create a log we can easily disable / enable and put it on all generators.